ipynb/wltests/sched-evaluation-full.ipynb

```diff
@@ -109,7 +109,7 @@
     "for test in collector.tests(workload='jankbench'):\n",
     "    logging.info(\"Results for: %s\", test)\n",
     "    collector.report(workload='jankbench', metric='frame_total_duration',\n",
-    "                     test=test, sort_on='99%', ascending=True)"
+    "                     test=\"^{}$\".format(test), sort_on='99%', ascending=True)"
```

Comment:
Don't you think that `libs/utils/wa_results_collector.py` should be fixed instead of this notebook ? Maybe both ?

Reply:
No, it's not a results collector issue.

In the API we already accept regexp for params such as `test`, the problem is that we do not pass in a regexp but just a substring... which is still a valid regexp but it matches both `list_view` and `image_list view`.

Reply:
Oh OK I just realized the `test` parameter of `wa_results_collector.boxplot()` is documented as a regex so you're right, it's fine like that :)